app/templates/src/main/java/package/service/_UserService.java

```diff
@@ -117,7 +118,17 @@ public User createUserInformation(String login, String password, String firstNam
         log.debug("Created Information for User: {}", newUser);
         return newUser;
     }
-
+    public User createUser(User user) {
+        String encryptedPassword = passwordEncoder.encode(RandomUtil.generatePassword());
+        user.setPassword(encryptedPassword);
+        user.setResetKey(RandomUtil.generateResetKey());
+        user.setResetDate(ZonedDateTime.now());
```

Comment:
I think you need to differentiate here between mongo/sql and cassandra. Cassandra requires a plain ``java.util.Date``